### DISCUSSION TITLE
In the first line of this book, "Who" is the "light of my life, fire of my loins"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#580] [Galaxy Trivia] [answered]
**Bonus:** Can you name the author?

**Comment by peterbaccaro** [comment]
**Lolita** and the author of the book was **Vladimir Nabokov**.

**Comment by thuongtruong1009** [comment, marked as answer]
The first line of the book you are referring to is from "Lolita" by Vladimir Nabokov, and in it, the protagonist Humbert Humbert refers to the young girl Dolores Haze (Lolita) as the "light of my life, fire of my loins."
